nibabel/freesurfer/io.py

```diff
@@ -160,6 +161,49 @@ def read_morph_data(filepath):
     return curv
 
 
+def write_morph_data(file_like, values, fnum=0):
+    """Write Freesurfer morphometry data `values` to file-like `file_like`
+
+    Equivalent to FreeSurfer's `write_curv.m`_
+
+    See also:
+    http://www.grahamwideman.com/gw/brain/fs/surfacefileformats.htm#CurvNew
+
+    .. _write_curv.m: \
+    https://github.com/neurodebian/freesurfer/blob/debian-sloppy/matlab/write_curv.m
+
+    Parameters
+    ----------
+    file_like : file-like
+        String containing path of file to be written, or file-like object, open
+        in binary write (`'wb'` mode, implementing the `write` method)
+    values : array-like
+        Surface morphometry values
+    fnum : int, optional
+        Number of faces in the associated surface
+    """
+    magic_bytes = np.array([255, 255, 255], dtype=np.uint8)
+
+    array = np.asarray(values).astype('>f4').squeeze()
```

Comment:
You could imagine that user was making a mistake, passing in a multidimensional array, that just happened to have only one dimension not length 1.   I mean, maybe you actually want to accept any of shape (N,), (1, N), (N, 1), but not other shapes.   As in `if array.ndim > 2 or (array.ndim == 2 and not 1 in array.shape):`

Actually maybe `array` is a bit confusing as name (looks like `np.array`) ...